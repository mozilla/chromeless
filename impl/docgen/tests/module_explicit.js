/**
 * In YUIDOC like fashion, it's completely allowable to have
 * the first comment block of a file provide module level
 * documentation.  One may also... 
 *
 * @module module_explicit_renamed
 *
 * @description
 * ... use the <at>description tag to explicitly provide module level
 * documentation, or they may combine both mechanisms (top commenting
 * (and desc tag).  In the latter case, docs will be concatenated
 * with a couple newlines.
 */
