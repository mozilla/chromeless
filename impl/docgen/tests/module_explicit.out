{
  "desc": "In YUIDOC like fashion, it's completely allowable to have\nthe first comment block of a file provide module level\ndocumentation.  One may also...\n\n... use the <at>description tag to explicitly provide module level\ndocumentation, or they may combine both mechanisms (top commenting\n(and desc tag).  In the latter case, docs will be concatenated\nwith a couple newlines.", 
  "filename": "tests/module_explicit.js", 
  "module": "module_explicit_renamed"
}
